circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
How about testing that what you installed actually starts and is available on the ports you specified?